dashboard-app/src/components/Charts/LineGraph.js

```diff
@@ -17,7 +17,13 @@ function LineGraph(props) {
     for (var dataObject of dataObjects) {
       var row = [];
       for (var key in dataObject) {
-        row.push(dataObject[key]);
+        let dataInstance = dataObject[key];
+        if (key === 'Time') {
+          // timestamp format is "2020-01-26T00:00:00.0+06:00Z"
+          dataInstance = dataInstance.substring(0, 10);
```

Comment:
reformat the date by accessing date and getting date objects https://www.w3schools.com/js/js_date_methods.asp look on the left tab for also creating date object